Permanent/ViewControllers/Profile/PublicProfilePageViewController.swift

```diff
@@ -249,51 +217,59 @@ extension PublicProfilePageViewController: UICollectionViewDataSource {
             let cell = collectionView.dequeueReusableCell(withReuseIdentifier: ProfilePageAboutCollectionViewCell.identifier, for: indexPath) as! ProfilePageAboutCollectionViewCell
             let shortDescriptionValue = viewModel?.blurbProfileItem?.shortDescription
             
-            cell.configure(shortDescriptionValue ?? viewModel?.archiveType.shortDescriptionHint)
+            if isEditDataEnabled {
```

Comment:
is this still necessary?

Reply:
I removed it, it's not necessary anymore.